mui/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@brightlayer-ui/icons-mui",
-    "version": "4.0.0",
+    "version": "3.3.0",
```

Comment:
Downgrading from 4.0.0 to 3.3.0?

Reply:
This was initially going to be a breaking change, but the logic introduced here keep it stable.